<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBLP Little Browser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/background.qrc">
    <normaloff>:/mouse.png</normaloff>:/mouse.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{background-image: url(:/background.jpg);}
#centralwidget{background-image: url(:/background.jpg);}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0">
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="4" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <widget class="QWidget" name="authorPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <widget class="QTableWidget" name="Table_1">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>1</y>
          <width>750</width>
          <height>500</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Title</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Author</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Modify Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Publisher</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Key</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="titlePage">
       <widget class="QLabel" name="Label">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>1</y>
          <width>641</width>
          <height>271</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="coauthorPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QTextBrowser" name="Browser">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>1</y>
          <width>671</width>
          <height>271</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="fuzzyPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QTableWidget" name="Table_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>711</width>
          <height>431</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Title</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Modify Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Key</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="LabelBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string>Search Byï¼š</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item alignment="Qt::AlignLeft">
        <widget class="QRadioButton" name="AuthorButton">
         <property name="text">
          <string>Author</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="TitleButton">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="CoauthorButton">
         <property name="text">
          <string>Coauthor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="KeyButton">
         <property name="text">
          <string>Key</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="FuzzyButton">
         <property name="text">
          <string>Fuzzy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="TextLine">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fuzzy">
        <property name="text">
         <string>Fuzzy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
        <property name="icon">
         <iconset resource="icons/background.qrc">
          <normaloff>:/search.png</normaloff>:/search.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="action_Language"/>
    <addaction name="action_Status"/>
   </widget>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>Function</string>
    </property>
    <addaction name="action_Clear"/>
    <addaction name="action_AuthorCount"/>
    <addaction name="action_FuzzyOpen"/>
    <addaction name="action_GroupAnalyze"/>
    <addaction name="action_KeyWordCount"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_HowToUse"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuFunction"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100,100, 200);</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Status"/>
   <addaction name="action_Clear"/>
   <addaction name="action_FuzzyOpen"/>
   <addaction name="action_AuthorCount"/>
   <addaction name="action_GroupAnalyze"/>
   <addaction name="action_KeyWordCount"/>
   <addaction name="action_HowToUse"/>
   <addaction name="action_Language"/>
   <addaction name="action_Exit"/>
  </widget>
  <action name="action_HowToUse">
   <property name="icon">
    <iconset resource="icons/background.qrc">
     <normaloff>:/IconParkHelp.png</normaloff>:/IconParkHelp.png</iconset>
   </property>
   <property name="text">
    <string>How To Use?</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/background.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="icons/background.qrc">
     <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="icon">
    <iconset resource="icons/background.qrc">
     <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="icon">
    <iconset>
     <normaloff>icons/author.png</normaloff>icons/author.png</iconset>
   </property>
   <property name="text">
    <string>Author of the Most Articles</string>
   </property>
  </action>
  <action name="action_AuthorCount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/background.qrc">
     <normaloff>:/author.png</normaloff>:/author.png</iconset>
   </property>
   <property name="text">
    <string>AuthorCount</string>
   </property>
  </action>
  <action name="action_FuzzyOpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/background.qrc">
     <normaloff>:/fuzzyopen.png</normaloff>:/fuzzyopen.png</iconset>
   </property>
   <property name="text">
    <string>FuzzyOpen</string>
   </property>
  </action>
  <action name="action_Language">
   <property name="icon">
    <iconset resource="icons/background.qrc">
     <normaloff>:/language.png</normaloff>:/language.png</iconset>
   </property>
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="action_GroupAnalyze">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/background.qrc">
     <normaloff>:/group.png</normaloff>:/group.png</iconset>
   </property>
   <property name="text">
    <string>GroupAnalyze</string>
   </property>
  </action>
  <action name="action_KeyWordCount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/background.qrc">
     <normaloff>:/keyword.png</normaloff>:/keyword.png</iconset>
   </property>
   <property name="text">
    <string>KeyWordCount</string>
   </property>
  </action>
  <action name="action_Status">
   <property name="icon">
    <iconset resource="icons/background.qrc">
     <normaloff>:/fuzzy.png</normaloff>:/fuzzy.png</iconset>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/background.qrc"/>
  <include location="icons/background.qrc"/>
 </resources>
 <connections/>
</ui>
